tools/src/test/scala/scala/scalanative/linker/MinimalRequiredSymbolsTest.scala

```diff
@@ -21,16 +21,16 @@ class MinimalRequiredSymbolsTest extends LinkerSpec {
   def isScala2_12 = ScalaNativeBuildInfo.scalaVersion.startsWith("2.12")
 
   @Test def default(): Unit = checkMinimalRequiredSymbols()(expected =
-    if (isScala3) SymbolsCount(types = 620, members = 3025)
-    else if (isScala2_13) SymbolsCount(types = 600, members = 3030)
-    else SymbolsCount(types = 690, members = 4185)
+    if (isScala3) SymbolsCount(types = 618, members = 3030)
+    else if (isScala2_13) SymbolsCount(types = 593, members = 3039)
+    else SymbolsCount(types = 690, members = 4187)
```

Comment:
I wonder why Process is even referenced in the first place for an empty main! That's definitely something worth to inspect to reduce minimal size of the binary. 
Of course not related the the PR

Reply:
alas, i don't know what this test does but i am assuming it has to do with `alloc`... why else? i hope i haven't copied incorrectly these values from the tests failing in previous iterations. (some numbers didn't change this time around they were smaller previously and not adjusted down.)